style.css

```diff
@@ -0,0 +1,138 @@
+.header {
+    background-color: rgba(11, 12, 12, 0.89);
+    width: 378px;
+    height: 92px;
+    position: absolute;
+  
+  }
+  body{
+  margin: 0%;
+  padding: 0%;
+  width: 378px;
+   }
+  /* contect info start */
+  .contect-info { 
+      width: 100%;
+      height: 7vh;
+  }
+  .contect-info span.fa { 
+    float: right;
+    margin-top: 2.5%;
+    padding-left: 1.3%;
+    color: aliceblue; 
+  }
+  .contect-info span.fa.fa-battery-full {
+    margin-right: 4%; 
+  }
+  .contect-info span.time { 
+    float: left;
+    margin-top: 2.5%;
+    margin-left: 4%; 
+    padding-left: 1.3%;
+    font-weight: bold;
+    color: aliceblue;
+  }
+    
+  /* contect info end */
+  
+  /* logo start */
+  .logo{ 
+    float: left;
+    margin: 0%;
+    padding: 0%;
+    margin-left: 5.2%; 
+    margin-bottom: 3.5%;
+    color: aliceblue;
+    font-family: 'Roboto' sans-serif;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+  }
+  .fa-bars{
+    float: right;
+    margin-top: 2.5%;
+    margin-right: 4.5%; 
+    padding-right: 1.3%;
+    font-weight: bold;
+    color: aliceblue;
+  }
+  
+  /* logo end */
+  
+   /* section start */
+  section.explain {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px; 
+  position: relative; 
+  height: 420px;
+  left: 0px;
+  top: 92px; 
+  background: #1C1A19;
+  
+  }
+  section.explain img{
+    
+    margin-left: 67.3%;
+    margin-top: -47%;
+   
```

Comment:
It would be better if you eliminate unnecessary spaces. if you don't it will cause linter error.